howto/android/access-instance.md

```diff
@@ -0,0 +1,151 @@
+(howto-access-instance)=
+# How to access an instance
+
+For certain scenarios, accessing an individual instance for debugging or development might be necessary. You can do this by:
+
+- Running `anbox-connect` for secure Android Debug Bridge (ADB) connections.
+- Exposing the ADB service upon Anbox instance creation.
```

Comment:
```suggestion
- Exposing the ADB service upon Anbox instance creation. We use the [`scrcpy`](https://github.com/Genymobile/scrcpy) tool for this method.
```

Reply:
Okay, I understood your point from the SU discussion. We can keep the classification to Not exposing the service (Anbox connect) vs Exposing ADB service.